paper/paper.md

```diff
@@ -0,0 +1,81 @@
+---
+title: 'pyOBIS: easy access to taxonomic occurrence records harvested from thousands of datasets'
+tags:
+  - Python
+  - oceanography
+  - marine data
+authors:
+  - name: Scott Chamberlain
+    equal-contrib: true
+  - name: Ayush Anand
+    equal-contrib: true
+    affiliation: 1
+  - name: Tylar Murray
+    corresponding: true
+    affiliation: 2
+  - name: Filipe Fernandes
+    corresponding: true
+  - name: Mathew Biddle
+    corresponding: true
+    affiliation: 3
+affiliations:
+ - name: National Institute of Technology Durgapur, India
+   index: 1
+ - name: IMaRS University of South Florida, US
+   index: 2
+ - name: National Oceanic and Atmospheric Administration, National Ocean Service, Integrated Ocean Observing System, US
+   index: 3
+date: 9 May 2023
+bibliography: paper.bib
+---
+
+# Summary
+The pyOBIS python package provides easy access to taxonomic occurrence records harvested from thousands of datasets.
+The package uses the API from the Ocean Biodiversity Information System (OBIS),
+a global open-access data and information clearinghouse on marine data for biodiversity for science, conservation,
+and sustainable development.
+Included in the pyOBIS package are built-in functions for accessing data on occurrences, taxon, nodes, checklists and datasets.
+The package provides easy export of data to Pandas DataFrame to help researchers focus more on analysis rather than data mining,
+and several included Jupyter notebooks demonstrate example analyses that can be used as a starting point for addressing research questions related to global and local distributions of species across space and time.
+Coupled together with other libraries like pyDwcViz,
+it forms an ecosystem of analysing Darwin Core Data with super ease through built-in functions.
+
+# Introduction
+
```

Comment:
@ocefpaf I have highlighted some of the possible use cases breifly here and will expand more in the `Why pyobis` section. Should we mention some more details here?